domains/mlsend2._domainkey.mail.satindar.json

```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Mailersend DKIM",
+  "owner": {
+    "username": "satindar31",
+    "email": "satindar31@tutanota.com"
+  },
+  "record": {
+    "CNAME": "mlsend2._domainkey.mailersend.net"
```

Comment:
The tests say that there's something wrong with this record but it looks fine to me.